### DISCUSSION TITLE
Possible to use application group container for caching?

### DISCUSSION THREAD

**Discussion by duc-ios** [onevcat/Kingfisher#1850] [Q&A]
I want to share the same ImageCache for my app and its extensions, is it possible?
FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: "group.bundle.id")

**Comment by onevcat** [comment]
Not quite sure but maybe it worths to try this before you setting an image:

```swift
guard let url = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: "group.bundle.id") else {
    return
}
let myCache = ImageCache(name: "my_cache", cacheDirectoryURL: url)
KingfisherManager.shared.cache = myCache
```
